hatch.toml

```diff
@@ -27,7 +26,7 @@ dependencies = [
 
 [envs.lint.scripts]
 black_check = [
```

Comment:
I doubt anyone is calling this directly, we can just remove from here and "checks" I think.

Unless does it need to be `ruff format --check` ? in which case, could update from `black_check` -> `fmt_check` ?